docs/csharp/language-reference/statements/jump-statements.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Jump statements - C# reference"
+description: "Learn about C# jump statements: break, continue, return, and goto."
+ms.date: 12/06/2021
+f1_keywords:
+  - "break_CSharpKeyword"
+  - "continue_CSharpKeyword"
+  - "return_CSharpKeyword"
+  - "goto_CSharpKeyword"
+helpviewer_keywords:
+  - "break statement [C#]"
+  - "break keyword [C#]"
+  - "continue statement [C#]"
+  - "continue keyword [C#]"
+  - "return statement [C#]"
+  - "return keyword [C#]"
+  - "goto statement [C#]"
+  - "goto keyword [C#]"
+---
+# Jump statements (C# reference)
+
+The following statements unconditionally transfer control:
+
+- The [`break` statement](#the-break-statement): terminates the closest enclosing [iteration statement](iteration-statements.md) or [`switch` statement](selection-statements.md#the-switch-statement).
+- The [`continue` statement](#the-continue-statement): starts a new iteration of the closest enclosing [iteration statement](iteration-statements.md).
+- The [`return` statement](#the-return-statement): terminates execution of the function in which it appears and returns control to the caller.
+- The [`goto` statement](#the-goto-statement): transfers control to a statement that is marked by a label.
```

Comment:
The [spec about jump statements](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/statements#jump-statements) also includes the `throw` statement in the list of jump statements. Should we cover `throw` in this article (and then to have the separate article only for `try` and its variants)?

Reply:
I would expect to see `throw` here, as it in my mind is a jump statement. But I defer to @BillWagner

Reply:
I thought about this for a while. There are good arguments on both sides.  In the end, I think it will be more clear to link to an article about `throw`, `try`, `catch` and `finally` (and probably exception filters) from this article than to cover `throw` here.

My reasoning is that `throw` is strongly related to `try`, `catch`, and `finally`. 

The `throw` statement is a jump statement in the spec. But what about the `throw` expression? It's not even a *statement*. If I were looking for `throw`, I'd expect to find information on both the *statement* and the *expression*. (Although, I think they will stay separate in the spec because of the 1st level headings)

Reply:
@BillWagner thank you; so, I've only added the link to the article about `throw` for now. (Later, I'll consolidate all exception-related statements into one article.)